tests/nightly/test_large_array.py

```diff
@@ -274,11 +265,40 @@ def test_diag():
     assert_almost_equal(r.asnumpy(), np.diag(a_np, k=k))
 
     # random k
-    k = np.random.randint(-min(LARGE_X, 64) + 1, min(h, w))
+    k = np.random.randint(-min(LARGE_X, SMALL_Y) + 1, min(LARGE_X, SMALL_Y))
     r = mx.nd.diag(a, k=k)
     assert_almost_equal(r.asnumpy(), np.diag(a_np, k=k))
 
 
+def random_2d_coordinates(x_low, x_high, y_low, y_high):
```

Comment:
It seems you are re-use the existing utility method: https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/test_utils.py#L410

Reply:
it is different and we need one value that to very large for tests case to be successful not any values between (1, very large value)

Reply:
will move this to test_utils.py